CMakeLists.txt

```diff
@@ -16,6 +16,13 @@ include(cmake/cli_input.cmake)
 include(cmake/extension.cmake)
 include(cmake/blob_path.cmake)
 
+if (NOT(DEFINED ENV{OPENSSL3_ROOT}))
+  message(FATAL_ERROR "OPENSSL3_ROOT not set")
```

Comment:
Are we mentioning anywhere how to fix this in case they encounter this error?

Reply:
yes mentioned in readme to export these variables